src/libYARP_sig/include/yarp/sig/IntrinsicParams.h

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright (C) 2006-2018 Istituto Italiano di Tecnologia (IIT)
+ * All rights reserved.
+ *
+ * This software may be modified and distributed under the terms of the
+ * BSD-3-Clause license. See the accompanying LICENSE file for details.
+ */
+
+#ifndef YARP_SIG_INTRINSICPARAMS_H
+#define YARP_SIG_INTRINSICPARAMS_H
+
+#include <yarp/os/Log.h>
+#include <yarp/os/Portable.h>
+#include <yarp/os/Property.h>
+#include <yarp/sig/api.h>
+
+namespace yarp {
+namespace sig {
+/**
+ * @brief The YarpDistortion enum to define the
+ * type of the distortion model of the camera.
+ */
```

Comment:
@Nicogene and probably @claudiofantacci  Do you have any reference for the details about this distortion models? There are probably obvious for vision experts, but it may be useful to have some kind of reference for both new users and as a formal reference to handle tricky corner cases.

Reply:
If there are some good documentation on this in other software project, those could also work as a reference.

Reply:
Even in software I would try to avoid citing Wipedia as much as possible (you never know if one day they will change the notation in the page). A slightly better idea is to cite a webpage (even a wikipedia page) using WebCite ( http://www.webcitation.org/ ).

Reply:
> you never know if one day they will change the notation in the page

Use permalinks: https://en.wikipedia.org/w/index.php?title=Distortion_(optics)&oldid=858291308 (shorter: https://en.wikipedia.org/w/index.php?oldid=858291308).

Reply:
@PeterBowman thanks, for wikipedia permalinks are definitely better than using webcite.

Reply:
Permalinks are ok. A more direct citation would have been OpenCV one, but that is by for more "unstable" than Wikipedia. Thanks @PeterBowman 👍

Reply:
That's exactly why I mentioned WebCite, so we can have a permalink-equivalent for OpenCV docs.